src/handlers/shared/start.ts

```diff
@@ -141,12 +141,11 @@ async function handleTaskLimitChecks(username: string, context: Context, maxConc
   // check for max and enforce max
 
   if (Math.abs(assignedIssues.length - openedPullRequests.length) >= maxConcurrentTasks) {
-    const log = logger.error(username === sender ? "You have reached your max task limit" : `${username} has reached their max task limit`, {
+    logger.error(username === sender ? "You have reached your max task limit" : `${username} has reached their max task limit`, {
```

Comment:
shouldn't it throw this so it gets posted as a comment?

Reply:
No because this is looping through teammates and checking each, instead I throw the error after this function stops looping so that we can check all.

So this is invoked at ~L60 above and I throw ~L70